<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ECATMain</class>
 <widget class="QMainWindow" name="ECATMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lim - EtherCAT Master</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="LE_IF_NAME">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>eth0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_SCAN">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="TW_SLAVE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>781</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="BTN_OP_MODE">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OP Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_DATA_EXCHANGE">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Exchange Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
